apps/src/templates/studioHomepages/teacherHomepageV2/sectionAvatars/PickerGrid.tsx

```diff
@@ -1,22 +1,62 @@
+import classNames from 'classnames';
 import React from 'react';
 
+import {COLORS, COLOR_LABELS, EMOJIS, EMOJI_LABELS} from './avatarConstants';
+
 import styles from './section-avatars.module.scss';
 
 interface PickerGridProps {
-  itemList: string[];
   type: 'emoji' | 'color';
+  selectCallback: (index: number) => void;
+  selected: number;
 }
 
-const PickerGrid: React.FC<PickerGridProps> = ({itemList, type}) => {
+const PickerGrid: React.FC<PickerGridProps> = ({
+  type,
+  selectCallback,
+  selected,
+}) => {
   const gridItems =
     type === 'emoji'
-      ? itemList.map((item, index) => (
-          <div key={index} className={styles.gridItem}>
+      ? EMOJIS.map((item, index) => (
```

Comment:
Most of this and the colors <div> are the same. Want to dedupe and extract to function?